Basic FE project template/ui-src/js/increase-number.js

```diff
@@ -0,0 +1,21 @@
+const counterAnim = (qSelector, start = 0, end, duration = 1000) => {
+  const target = document.querySelector(qSelector);
+  let startTimestamp = null;
+  const step = (timestamp) => {
+    if (!startTimestamp) startTimestamp = timestamp;
+    const progress = Math.min((timestamp - startTimestamp) / duration, 1);
+    target.innerText = Math.floor(progress * (end - start) + start);
+    if (progress < 1) {
+      window.requestAnimationFrame(step);
+    }
+  };
+
+  window.requestAnimationFrame(step);
+};
+
+document.addEventListener("DOMContentLoaded", () => {
+  counterAnim(".count1", 0, 1200, 2000);
```

Comment:
Here you can create a function named triggerAnimations() that can contain the 4 counterAnim functions you have here. Instead of triggering the new triggerAnimations function on DOMContentLoaded, you need to create a scroll function similar to the model I sent you.